protected/controllers/SiteController.php

```diff
@@ -476,6 +483,15 @@ public function actionFeed(){
         exit;
     }
 
+    public function formatBytes($bytes, $precision = 2) {
+        $units = array('B', 'KB', 'MB', 'GB', 'TB');
+
+        $base = log($bytes, 1024);
+        $suffixes = array('', 'K', 'M', 'G', 'T');
```

Comment:
@only1chunts When you look at how the volume of data is displayed in the home page, the units of data volume is displayed twice so currently on http://gigadb.org, it shows 43 T then underneath it also says `Data volume(TB)`. Perhaps it would be better just to display a number for the volume of data then underneath it could say `terabytes of data` so it is consistent with the numbers of datasets, samples and files.

Reply:
The T after 43T is not required.
Words under the number 43 should be "Data volume (TB)"
The inclusion of commas every 3 decimal places would be good.